plugin/src/main/kotlin/io/wttech/gradle/config/prop/StringProp.kt

```diff
@@ -66,6 +70,7 @@ class StringProp(group: Group, name: String) : Prop(group, name) {
 
     private var valueSaved: (String?) -> Any? = { v ->
         when (valueType.get()) {
+            ValueType.PATH -> v?.let { v.replace("\\", "/")}
```

Comment:
I don't remember if that code was effectively used to be saved in gradle.properties; if does it's ok